test/shared/src/main/scala/zio/test/TestAspect.scala

```diff
@@ -51,6 +55,19 @@ object TestAspect {
       def apply[R >: R0 <: R0, E >: E0 <: E0](test: ZIO[R, E, TestResult]): ZIO[R, E, TestResult] = f(test)
     }
 
+  /**
+   * An aspect that retries a test until success, without limit.
```

Comment:
I can compose this with a limit?

Reply:
Sure, e.g. `retry >>> timeout(60.seconds)`, but if you want something fancier, you're better off using `repeat(schedule)`.

Reply:
What I figured, just wanted to make sure :)

Reply:
You know how I _LOVE_ composability. 😉